tests/integration/advanced/test_timeseries.py

```diff
@@ -44,38 +43,52 @@ def test_0_grouped_regression_timeseries(self):
             test = test.append(subframe[int(length * train_ratio):])
 
         target = 'Traffic'
-        nr_preds = 2
         order_by = 'T'
 
         # Test multiple predictors playing along together
-        a = {}
-        a[1] = predictor_from_problem(pd.read_csv('tests/data/hdi.csv'), ProblemDefinition.from_dict(
-            {'target': 'Development Index', 'time_aim': 10}))
-        a[1].learn(pd.read_csv('tests/data/hdi.csv'))
-        a[1].predict(pd.read_csv('tests/data/hdi.csv').iloc[0:10])
-
-        prdb = ProblemDefinition.from_dict({'target': target,
-                                            'time_aim': 30,
-                                            'nfolds': 10,
-                                            'anomaly_detection': True,
-                                            'timeseries_settings': {
-                                                'use_previous_target': True,
-                                                'group_by': ['Country'],
-                                                'nr_predictions': nr_preds,
-                                                'order_by': [order_by],
-                                                'window': 5
-                                            }
-                                            })
-
-        a[2] = predictor_from_problem(train, prdb)
-
-        a[2].learn(train)
-        preds = a[2].predict(test)
+        pred_arr = {}
```

Comment:
I wonder if keeping both tests here is still desired. May be better if we test the "multiple predictors in parallel" someplace else?

Reply:
Oh fuck,  100% remove this test it was a temp thing, shouldn't have been kept there.

Reply:
Actually, I'll keep both as separate tests, to increase coverage (in particular, test TS without anomalies, autoregressiveness, and single-step forecasting)